tests/countries/test_saudi_arabia.py

```diff
@@ -173,19 +173,19 @@ def test_hijri_based_with_two_holidays_in_one_year(self):
     def test_l10n_default(self):
         self.assertLocalizedHolidays(
             ("2023-02-22", "يوم التأسيسي"),
-            ("2023-04-21", "(تقدير) عطلة عيد الفطر"),
-            ("2023-04-22", "(تقدير) عطلة عيد الفطر"),
-            ("2023-04-23", "(تقدير) عطلة عيد الفطر"),
-            ("2023-04-24", "(تقدير) عطلة عيد الفطر"),
-            ("2023-04-25", "(تقدير ملاحظة) عطلة عيد الفطر"),
-            ("2023-04-26", "(تقدير ملاحظة) عطلة عيد الفطر"),
-            ("2023-06-27", "(تقدير) يوم عرفة"),
-            ("2023-06-28", "(تقدير) عطلة عيد الأضحى"),
-            ("2023-06-29", "(تقدير) عطلة عيد الأضحى"),
-            ("2023-06-30", "(تقدير) عطلة عيد الأضحى"),
-            ("2023-07-02", "(تقدير ملاحظة) عطلة عيد الأضحى"),
+            ("2023-04-21", "عطلة عيد الفطر (المقدرة)"),
+            ("2023-04-22", "عطلة عيد الفطر (المقدرة)"),
+            ("2023-04-23", "عطلة عيد الفطر (المقدرة)"),
+            ("2023-04-24", "عطلة عيد الفطر (المقدرة)"),
+            ("2023-04-25", "عطلة عيد الفطر (المقدرة ملاحظة)"),
```

Comment:
```suggestion
            ("2023-04-25", "عطلة عيد الفطر (المقدرة، ملاحظة)"),
```
and same for the line below this and other places